packages/cdk8s-cli/templates/java-app/help

```diff
@@ -0,0 +1,12 @@
+========================================================================================================
+
+ Your cdk8s Java project is ready!
+
+   cat help      Prints this message  
+   cdk8s synth   Synthesize k8s manifests to dist/
```

Comment:
If compile _is_ needed, then add it too

Reply:
Would it make more sense to add it to `cdk8s.yaml `?

```
language: java
app: *mvn compile* && mvn exec:java -Dexec.mainClass="com.mycompany.app.HelloKube"
imports:
  - k8s 
```

Reply:
Yeah maybe... Not sure. I think if you work with an IDE the IDE will continuously compile for you so that's not really required but it couldn't hurt too much I guess.

Reply:
I'll just let it be, I think it's fine as-is

Reply:
Can you please add `mvn compile` to `help`?

Reply:
Done!